### ISSUE TITLE
Model download link invalid

### ISSUE THREAD

**Issue by Loucy-28** [zhyever/Monocular-Depth-Estimation-Toolbox#103]
Hi, thank you for your great work. I'm trying to reproduce the BinsFormer (Swin-L on NYU) results, but the Google Drive link for the pre-trained model seems to be broken. Could you please provide an updated link? Thanks!

**Comment by zhyever** [comment]
Sorry guys, I deleted the file on my google drive by mistake and cannot recover it now... I think I can re-train the model at the end of July and release it in August. Thanks a lot for your interest to this repo and my work. (this issue is also mentioned in #102)

**Comment by Loucy-28** [comment]
Mistakes happen. Thanks for keeping us updated—looking forward to the release in August.

**Comment by zhyever** [comment]
I found local backups for binsformer. Please check:

swinl: https://drive.google.com/file/d/1sZFsdU-q8QTmfMXeL5yLTHSPKl8ls6H6/view?usp=sharing

swint: https://drive.google.com/file/d/1bHgOnNcNLlQyyzFiBZao280cMjz7mhMy/view?usp=sharing

**Comment by Loucy-28** [comment]
Thanks for sharing the pre-trained models! I’m trying to train Binsformer on Gated2Depth data(using 3 stacked gated images instead of RGB). The loss decreases, but evaluation metrics stay poor (see attached log [20250721_182441.log](https://github.com/user-attachments/files/21381893/20250721_182441.log)).
Tried:
Using your pre-trained model
Switching to L1 loss
How should I modify it?

**Comment by zhyever** [comment]
Hi, after checking the log, I don't think the model is converging. 

Please first check if the `max_depth` is correctly set to the max depth of Gated2Depth dataset.

Visualization of results might be necessary.

If it's still not converging, please try to re-run the code multiple times with different seeds. Sometimes, it can work.

**Comment by Loucy-28** [comment]
max_depth is correct, how to set different seeds?

**Comment by zhyever** [comment]
Please check: https://github.com/zhyever/Monocular-Depth-Estimation-Toolbox/blob/d8aff50e75c4b0c183361c7690959d0269b45a9d/tools/train.py#L44

Simply adding --seed xxx can work. 

Given your loss is around 2.8, I guess a good run will reduce the loss to around 1.0 quickly (in the first several steps). If the loss is still around 2.8, simply kill and re-run.

**Comment by Loucy-28** [comment]
Okey,I'll try.

**Comment by Loucy-28** [comment]
Hi,the previous issue with the model not converging has been resolved, but I've run into a new problem: how can I visualize the color depth map output by the model? I used the commands in your inference.md file:

python tools/test.py configs/binsformer/binsformer_swinl_22k_w7_kitti.py \
    work_dirs/binsformer_swinl_22k_w7_kitti/20250728_133435/iter_38400.pth \
    --show

But the output isn't what I expected.(

<img width="1216" height="352" alt="Image" src="https://github.com/user-attachments/assets/247ab6c3-62ba-48c1-9a21-158d2bd6e7a9" />

) Did I make a mistake somewhere?How should I output the color depth map?

**Comment by Loucy-28** [comment]
Already modified
